test/core/iomgr/tcp_server_posix_test.cc

```diff
@@ -59,7 +60,7 @@
 #include "src/core/lib/resource_quota/api.h"
 #include "test/core/test_util/port.h"
 
-#define LOG_TEST(x) gpr_log(GPR_INFO, "%s", #x)
+#define LOG_TEST(x) LOG(INFO) << #x
```

Comment:
nit: can we remove this macro? I see there's no good point now that we have LOG(INFO),